spec/crawler_spec.rb

```diff
@@ -0,0 +1,22 @@
+require_relative '../script'
+
+RSpec.describe Crawler do
+	subject(:crawler) { Crawler.new(document_collection, url_collection)}
+
+	# let(:url) {"/wikia/dragon_age_2"}
+	# let (:url_fetcher){UrlFetcher.new}
+	# let (:content) {url_fetcher.fetch(:url)}
+	# let(:document) {Document.new(:url, :content)}
+	let(:document_collection) {DocumentCollection.new}
+	let(:url_collection) {BreadthFirstUrlCollection.new}
```

Comment:
I think you removed `BreadthFirstUrlCollection`, so this shouldn't be necessary.